File: kraken/src/main/java/glitch/kraken/object/json/Game.java
package glitch.kraken.object.json;

import glitch.api.objects.json.interfaces.IDObject;
import lombok.Data;

@Data
public class Game implements IDObject<Long> {
    private final Long id;
    private final Image box;
    private final Long gigantbombId;
    private final Image logo;
    private final String name;
    private final Long popularity;
}

File: kraken/src/main/kotlin/glitch/kraken/object/json/Game.kt
package glitch.kraken.`object`.json

import glitch.api.objects.json.interfaces.IDObject

/**
 *
 * @author Damian Staszewski [damian@stachuofficial.tv]
 * @version %I%, %G%
 * @since 1.0
 */
class Game(
        override val id: Long,
        val box: Image,
        val gigantbombId: Long,
        val logo: Image,
        val name: String,
        val popularity: Long
) : IDObject<Long>
